totalRP3/Core/Vanilla/CompanionUtilOverrides.lua

```diff
@@ -0,0 +1,322 @@
+-- This should be converted to an [AllowLoadGameType ...] directive when supported
+-- on all clients.
+--
+-- While the collections journal wasn't introduced until Mists, it was added
+-- into Cataclysm and then Wrath (CN) as part of Classic.
+
+if LE_EXPANSION_LEVEL_CURRENT >= LE_EXPANSION_WRATH_OF_THE_LICH_KING then
+	return;
+end
```

Comment:
At some point I'll get to the TODO in Core.xml to move all our loading to the TOC.

When I do that, I'll remove this logic and make the client skip the file via `[AllowLoadGameType standard, mists, cata, wrath]` entirely.